Specification.md

```diff
@@ -172,18 +182,40 @@ but what is allowed by this specification.
 There MUST appear one “blank line” between subsequent *records*;
 additional “blank lines” MAY appear.
 
-The *records* don’t have to appear in any order.
-There MAY exist multiple *records* for the same day.
-These are treated as distinct.
-
-### Technical remarks
+*Records* MAY appear in any order in the file.
 
-The file extension MUST be `.klg`, e.g. `times.klg`.
+There MAY exist multiple *records* with the same date.
+These MUST be treated as distinct.
 
+The file extension SHOULD be `.klg`, e.g. `times.klg`.
 The file encoding MUST be UTF-8.
 
-Newlines MUST be encoded with the LF linefeed character (escape sequence `\n`).
-There SHOULD be a “newline” at the end of the file.
+Newlines MUST be encoded with either the
+linefeed character (LF, escape sequence `\n`),
+or carriage return and linefeed character (CRLF, escape sequences `\r\n`).
+These two styles SHOULD NOT be mixed within the same file.
+
+There SHOULD be a newline at the end of the file.
+
+## III. Evaluating data
+
+### Total time
+The resulting *total time* of a record MUST be computed by summing up its *entries*:
+positive values add to the *total time*,
+negative values deduct from it.
+The resulting *total time* MAY be 0;
+it MAY be negative;
+it MAY be greater than 24 hours.
+
+Overlapping *ranges* MUST be counted individually.
+(They MUST NOT be merged together.)
```

Comment:
👍 for simplifying the sentences.

I also don’t like the term `merge together`, because I think a) `together` is superfluous and maybe a bit weird here, and b) it would be better to say `offset against each other`. What I’m trying to say is that `12:00-13:00` and `12:30-13:30` results in `2h` and not in `1h30m`. Maybe it would help to add this as example:

> Overlapping *ranges* MUST be counted individually and MUST NOT be offset against each other.
> E.g., the two entries `12:00-13:00` and `12:30-13:30` result in total time of `2h`.

Reply:
👍 That looks good, and the reasoning is very valid to me.

Reply:
https://github.com/jotaen/klog/pull/88/commits/316f12d04a6a4258651cb67812389f9a64645ac2